### ISSUE TITLE
Feature request: Support woff2

### ISSUE THREAD

**Issue by kookma** [Jermolene/TiddlyWiki5#4538]
Tiddlywiki 5.1.22 nicely shows the imported web font file (.woff) as tiddler with _application/font-woff_ type
and when opened or edited, a yellow rectangle showing "This tiddler contains binary data"

I use web fonts of version 2 called woff2 in this way to customize Tiddlywiki, I noticed TW does not recognize woff2, so I may request to add these two features

- Add support to show imported .woff2 as _application/font-woff2_ and recognize them as binary file
- Add support to use such file with _datauri_ like below as embedded font.

```
@font-face {
  font-family: 'Roboto';
  font-style: normal;
  font-weight: 300;
  src: local("Roboto"), url(<<datauri "roboto-v20-latin-300.woff2">>) format("woff2");
}
```

**Comment by Jermolene** [comment]
Thanks @kookma, fix upcoming. Let me know how it goes.

**Comment by Jermolene** [comment]
Fixed in c7d8492842844d7de12b7a327f978ff114708d84

It'll take 10 minutes for the fix to appear at https://tiddlywiki.com/prerelease

**Comment by kookma** [comment]
@Jermolene 
I tested some web fonts like Inconsolata from Google font (.woff2) and they work great on the new TW 5.1.22pre. The woff2 are small in size and now I can create and distribute Tiddlywiki with beautiful and customized fonts 

Wonderful! A million thanks.
